specification/batch/data-plane/Microsoft.Batch/stable/2018-12-01.8.0/BatchService.json

```
@@ -13380,11 +13383,13 @@
         "maxTasksPerNode": {
           "type": "integer",
           "format": "int32",
-          "title": "The maximum number of tasks that can run concurrently on a single compute node in the pool."
+          "title": "The maximum number of tasks that can run concurrently on a single compute node in the pool.",
+          "description": "The default value is 1. The maximum value is 4 times the number of cores of the vmSize of the pool, or 256, whichever is smaller."
```

```suggestion
          "description": "The default value is 1. The maximum value is the smaller of 4 times the number of cores of the vmSize of the pool or 256"
```
I don't really like the two commas. Either this or get rid of the comma in 'vmSize of the pool or 256'